README.md

```diff
@@ -1,75 +1,19 @@
-project-template
-==============================
+# Optimal Stopping Point
 
-template for the team to use
+The aim of this tool is to predict an optimal stopping point for CI tests based on their test duration (runtimes). We perform an initial data analysis as well as feature engineering on the CI test data. Furthermore, we also calculate the optimal stopping point by identifying the distribution of the test duration values for different CI tests and comparing the distributions between the passing and failing tests.
```

Comment:
What feature engineering is done on the raw data? I thought we were collecting and training on run duration.

Reply:
Yeah, we are collecting and training on test duration, but we also try to evaluate various distributions to find the best fit and compare the p values before finalizing what model we want to use. All of this would be a part of this feature engineering process. If you think this name isn't suitable we can call it "Fit distribution" or "Finding best distribution" also.

Reply:
I would not call determining the best fit distribution to be feature engineering. I thought feature engineering was more along the lines of transforming the raw inputs in some way before applying them  to the model so it can learn the underlying task better.

Maybe you could say that you are performing "automated model selection"?

Reply:
I like something on the lines of "Finding best distribution"

Reply:
Are we training a model? https://github.com/redhat-et/optimal-stopping-point-tool/pull/4#discussion_r1094935509